src/indev/lv_indev.c

```diff
@@ -1275,6 +1271,37 @@ static void indev_proc_release(lv_indev_t * indev)
     }
 }
 
+static void indev_proc_pointer_diff(lv_indev_t * indev)
+{
+    lv_obj_t * obj = indev->pointer.last_pressed;
+    if(obj == NULL) return;
+    if(indev->pointer.diff == 0) return;
+
+    indev_obj_act = obj;
+
+    bool interactive = lv_obj_is_group_def(obj);
+
+    if(interactive) {
+        uint32_t k = indev->pointer.diff < 0 ? LV_KEY_LEFT : LV_KEY_RIGHT;
+        uint32_t cnt = LV_ABS(indev->pointer.diff);
+        while(cnt) {
+            send_event(LV_EVENT_KEY, &k);
```

Comment:
Should we define a new event and send the diff directly. Only in this way, we can make sure every turn of wheel is counted in.

Reply:
In the read callback or [the event](https://github.com/lvgl/lvgl/pull/5057/files#diff-63956c41f2f8adef0b3fa8fb8d8621c777559f0aeb54f42388e68895d3c15fcdR195) you can set the diff to 3 for example.

Reply:
It may be more appropriate to pack the diff together and send a total number.

Reply:
It's doable, but it requires to update all the widget receiving KEY events. I suggest merging it as it is and add support the diff cunt event parameter in a new PR later.

Reply:
we need a new scroll event to distinguish its origin from the crown device.